<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="icon">
    <iconset resource="antennaldo.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="antennaldo.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤销(Z)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>恢复(Y)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有窗口</string>
   </property>
   <property name="toolTip">
    <string>关闭所有窗口</string>
   </property>
  </action>
  <action name="tileAction">
   <property name="text">
    <string>平铺</string>
   </property>
   <property name="toolTip">
    <string>平铺</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>重叠</string>
   </property>
   <property name="toolTip">
    <string>重叠</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>上一个</string>
   </property>
   <property name="toolTip">
    <string>上一个</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个</string>
   </property>
   <property name="toolTip">
    <string>下一个</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="searchAction">
   <property name="icon">
    <iconset resource="antennaldo.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="toolTip">
    <string>搜索</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="antennaldo.qrc"/>
 </resources>
 <connections/>
</ui>
